src/bus/can_socket-rtcan.cpp

```diff
@@ -64,21 +64,20 @@ struct can_handle {
 };
 }
 
-double printCycle = 0.1;
 double startTime = highResolutionSystemTime();
 // std::stringstream sstm;
 // sstm << "/home/robot/log_" << startTime << ".log";
 std::string filename = "/home/robot/log_" + boost::lexical_cast<std::string>(startTime);;
```

Comment:
I don't think we need these variables here (we can delete the logging and stack trace stuff from the constructor too).